README.md

```diff
@@ -87,5 +87,16 @@ examplepackage/
     └── share
 ```
 
+## Contributing
+Clone this repository, create a virtual environment, then install dependencies with:
+```
+pip install . .[dev]
```

Comment:
Note: remember on macOS this may fail without quotes :-) Also I'd suggest adding the `-e` (editable) flag.

So:

```suggestion
pip install -e ".[dev]"
```